pontoon/base/static/css/style.css

```diff
@@ -1237,3 +1237,7 @@ body > .container {
   font-style: italic;
   margin-top: 20px;
 }
+
+.loading {
+  display: none;
+}
```

Comment:
Move to translate.css and make selector more specific, e.g.  `#entitylist .loading`.